README.md

```diff
@@ -1 +1,199 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+ 
+  <br/>
+
+  <h3><b>Hello Microverse README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#HTML-CSS)
+    - [Tech Stack](#HTML-CSS)
+    - [Key Features](#Header)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello-Microverse] <a name="Hello Microverse"></a>
+
+
+
+**[Hello-Microverse]** is a page with the heading hello microverse in my favorite color
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="html-css"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Markup</summary>
+  <ul>
+    <li><a href="html.com">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Style</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+
+
+### Key Features <a name="Header"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+### Install
+
+Install this project with:
+
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Gachuna)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/victor-wabuko-02ab9b255)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
```

Comment:
- [ ]  I think it would be better to add the future feature in your README to replace the default generated  placeholders as suggested by the previous Reviewer 😎😎